pkg/scheduler/util/topologies.go

```diff
@@ -48,6 +50,18 @@ func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, namespaces sets.String, se
 	return true
 }
 
+func podMatchesTermsNamespceWithRegexp(pod *v1.Pod, namespaces sets.String) bool {
+	if namespaces.Has(pod.Namespace) {
+		return true
+	}
+	for ns := range namespaces {
+		if match, _ := regexp.MatchString(ns, pod.Namespace); match {
```

Comment:
I know that more significant changes have been suggested in a different review, so I'm not sure if this will be valid, but we shouldn't be silently ignoring the error returned by MatchStrings here.